.huskyrc

```diff
@@ -1,5 +1,5 @@
 {
   "hooks": {
-    "pre-commit": "lint-staged"
+    "pre-commit": "lint-staged && yarn lerna run build --since next"
```

Comment:
`next` won't be stable here. Is there a solution here? https://stackoverflow.com/questions/3161204/how-to-find-the-nearest-parent-of-a-git-branch

Reply:
I'm not sure we can inject this value since we're in an `rc` file?

We already hard-code this value to tell lerna the release branch. We would only have to update this once we move it to `master`.